### ISSUE TITLE
Help entry for `--disable-cleanup-builddir`

### ISSUE THREAD

**Issue by cwant** [easybuilders/easybuild-framework#3150]
`eb` has an option `--cleanup-builddir` which has the following help text (`--help`):

```
--cleanup-builddir  Cleanup build dir after successful installation. (def True)
```

But I didn't want to clean that directory, so I tried `--cleanup-builddir=False`, which produced the error:

```
eb: error: --cleanup-builddir option does not take a value
```

So I tried `--no-cleanup-builddir`, which wasn't a valid option.

So I then grep-ed the sources, where I found `--disable-cleanup-builddir` as the correct option to use.

Long story short: the software should list `--disable-cleanup-builddir` in the text produced by `--help` (particularly since the only flag listed in the help related to this is the default).

**Comment by boegel** [comment]
fixed in #3151

**Comment by cwant** [comment]
Thanks! 👍
